internal/core/src/storage/FileManager.h

```diff
@@ -28,6 +28,26 @@
 
 namespace milvus::storage {
 
+struct FileManagerContext {
+    FileManagerContext() : chunkManagerPtr(nullptr) {
+    }
+    FileManagerContext(const FieldDataMeta& fieldDataMeta,
+                       const IndexMeta& indexMeta,
+                       const ChunkManagerPtr& chunkManagerPtr)
+        : fieldDataMeta(fieldDataMeta),
+          indexMeta(indexMeta),
+          chunkManagerPtr(chunkManagerPtr) {
+    }
+    bool
+    Valid() const {
+        return chunkManagerPtr != nullptr;
+    }
+
+    FieldDataMeta fieldDataMeta;
```

Comment:
why not use pointer 🤔？

Reply:
This is a config object transferred from outside. It is a little strange to use it with pointer.